config/manager/manager.yaml

```diff
@@ -22,18 +22,35 @@ spec:
       labels:
         control-plane: controller-manager
     spec:
+      securityContext:
+        runAsNonRoot: true
       containers:
       - command:
         - /manager
         args:
-        - --enable-leader-election
+        - --leader-elect
```

Comment:
I cannot find the option in the controller-manger code.
Is this necessary?

Reply:
This option was added by kubebuilder and we didn't edit it and don't use this YAML